docs/src/token-2022/extensions.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Extension Guide
```

Comment:
Is the plan to have this guide on usage of each of the different extensions within token-2022? Are these added to the CLI?

Reply:
Correct, and no, they aren't added to the CLI yet, just JS